README_cmake.md

```diff
@@ -948,6 +940,30 @@ Python 3.x. The result will be a a shared library of the form
 `<target>_pystub.<soabi>.so`, where <soabi> describes the specific Python version and platform (e.g., `cpython-310-darwin` for Python 3.10 on OSX.) See
 `README_python.md` for examples of use.
 
+#### `add_halide_python_extension_library`
+
+This function wraps the outputs of one or more `add_halide_library` targets with glue code to produce
+a Python Extension library.
+
+```
+add_halide_python_extension_library(
+    target
+    [MODULE_NAME module-name]
+    HALIDE_LIBRARIES library1 ...
+)
+```
+
+The `MODULE_NAME` argument specifies the name of the Python module that will be created. If omitted,
+it defaults to `target`.
+
+`HALIDE_LIBRARIES` is a list of one of more `add_halide_library` targets. Each will be added to the
+extension as a callable method of the module. Note that every library specified must be built with
+the `PYTHON_EXTENSION` keyword specified, and all libraries must use the same Halide runtime.
+
+The result will be a a shared library of the form
+`<target>.<soabi>.so`, where <soabi> describes the specific Python version and
+platform (e.g., `cpython-310-darwin` for Python 3.10 on OSX.)
```

Comment:
nit: it's "macOS" now. El Capitan was the last version to use the "OS X" branding.

Reply:
...tell that to `Target::OS::OSX` :-)

Reply:
We should probably have an alias for that tqbh :-)